src/queues.c

```diff
@@ -183,30 +183,29 @@ int queues_notification_close_id(int id, int reason)
 {
         notification *target = NULL;
 
-        for (GList *iter = g_queue_peek_head_link(displayed); iter;
+        for (GList *iter = g_queue_peek_head_link(displayed);
+             iter && !target;
```

Comment:
What's the point of this change? All it's doing it adding a bit of overhead because now we have to check `target` on each iteration and hurts readability.